pages/el-otro.js

```diff
@@ -39,13 +45,22 @@ function Home(props) {
 
 Home.getInitialProps = async function(context) {
   let query = context.query.q;
+  // set up multi language support
+  stemmerSupport(lunr);
+  multi(lunr);
+  es(lunr);
+  lunr.multiLanguage('en', 'es');
+  
   if (query) {
-    let normalizedQuery = query.toLowerCase();
-    let results = data.filter((result) =>
-      result.text.includes(normalizedQuery)
-    );
+    const normalizedQuery = unidecode(query.toLowerCase());
+    const idx = lunr.Index.load(elOtroIndex);
```

Comment:
@froi Can this be done at the module's top level, just once?

Reply:
I would imagine so. I’ll test it.

Reply:
Done.